CMakeLists.txt

```diff
@@ -286,13 +286,54 @@ IF (KokkosKernels_INST_OFFSET_SIZE_T)
   LIST(APPEND OFFSET_LIST "size_t")
 ENDIF()
 
+# ==================================================================
+# Enable Third Party Libraries
+# ==================================================================
+
+IF (TPL_ENABLE_BLAS)
+  SET(KOKKOSKERNELS_ENABLE_TPL_BLAS ${TPL_ENABLE_BLAS})
```

Comment:
Do you intend for these variables to be visible outside of this package?  If so, you'll need to use `GLOBAL_SET` or something like that.

Reply:
No these variables are internal only.